definitions/checks/system_registration.rb

```diff
@@ -5,15 +5,19 @@ class Checks::SystemRegistration < ForemanMaintain::Check
     tags :default
 
     confine do
-      file_exists?('/etc/rhsm/rhsm.conf') &&
-        !feature(:foreman_server) &&
-        feature(:foreman_proxy)
+      file_exists?('/etc/rhsm/rhsm.conf')
     end
+
+    message <<-MESSAGE
+      https://access.redhat.com/documentation/en-us/red_hat_satellite/6.3-beta/html/installation_guide/preparing_your_environment_for_installation#system_requirements
```

Comment:
Since these are Beta docs, those links will not be valid after GA release. 
@swapab, you need to change those links.

Reply:
Yes. Above links won't work since now 6.3 is GA